MSCA_application.csl

```diff
@@ -0,0 +1,1256 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="never" page-range-format="chicago">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Marie Skłodowska-Curie Actions</title>
+    <title-short>Marie Curie fellowship</title-short>
+    <id>http://www.zotero.org/styles/marie-skłodowska-curie-actions</id>
+    <link href="http://www.zotero.org/styles/marie-skłodowska-curie-actions" rel="self"/>
+    <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
```

Comment:
Is there some proper documentation to this reference style? Or can you upload something at least? This is also important for the naming of the style.